packages/chakra-ui-docs/pages/skeleton.mdx

```diff
@@ -0,0 +1,55 @@
+import SEO from "../components/SEO";
+
+<SEO
+  title="Skeleton"
+  description="Skeleton is used to display the loading state of some component"
+/>
+
+# Skeleton
+
+Skeleton is used to display the loading state of some component.
+
+## Import
+
+```js
+import { Skeleton } from "@chakra-ui/core";
+```
+
+## Usage
+
+You can use it as a standalone component
+
+```jsx
+<div>
+  <Skeleton height="20px" my="10px" />
+  <Skeleton height="20px" my="10px" />
+  <Skeleton height="20px" my="10px" />
+</div>
+```
+
+Or to wrap another component to take the same height and width
+
+```jsx
+<Skeleton>
+  <div>contents wrapped</div>
+  <div>won't be visible</div>
+</Skeleton>
+```
+
+### Skeleton color
+
+You can change the animation color with `colorStart` and `colorEnd`.
+
+```jsx
+<Skeleton colorStart="pink" colorEnd="orange" height="20px" />
+```
+
+### Skipping the skeleton when content is loaded
+
+You can prevent the skeleton from rendering using the `isLoaded` prop
+
+```jsx
+<Skeleton isLoaded>
+  <span>Chakra ui is cool</span>
+</Skeleton>
+```
```

Comment:
To keep the docs as relatable as possible, I thought it might be great to add a data-fetching example here. Just like I showed below.

Maybe even use a sample card from a popular site like Airbnb. Thank you